feat: Adding new input command for enabled or not job summary publishment (#721)
cypress-io/github-action: README.md

- [timeouts](#timeouts) to avoid hanging CI jobs
- [print Cypress info](#print-cypress-info) like detected browsers
- [run tests nightly](#nightly-tests) or on any schedule

> - [Publish tests summary](#publish-tests-summary)

Reviewer: ```suggestion
- [Suppress test summary](#suppress-test-summary)
```

The table of contents needs to agree with the content title.

[![cron example](https://github.com/cypress-io/github-action/workflows/example-cron/badge.svg?branch=master)](.github/workflows/example-cron.yml)

> ### Publish tests summary

Reviewer: Change to "Suppress test summary"
due to decision to keep the default that summaries are published.

> Sometimes users want to publish the test's summary to the pull request even if the current runtime environment does not support job summaries. 
> For example, a user running Cypress tests using a matrix and wanting to retrieve the tests summary for each matrix job.

Reviewer 2: ```suggestion
Sometimes users wants to publish tests to summary using a specific action. 
For example, a user running Cypress tests using a matrix and wanting to retrieve the tests summary for each matrix job and use a specific action that merges reports.
```

```yml

> name: info

Reviewer: Use meaningful name, for example:
`name: Example no summary`

> name: info

Reviewer 2: ```suggestion
name: Example no summary
```

> on: [push]

Reviewer: Remove brackets
```yml
on: push
```

> on: [push]

Reviewer 2: ```suggestion
on: push
```

jobs:
  cypress-run:
    runs-on: ubuntu-22.04
    steps:
      - name: Checkout
        uses: actions/checkout@v3
      - name: Cypress run
        uses: cypress-io/github-action@v5
        with:

>           publish-summary: true

Reviewer: Change to
`publish-summary: false`

to follow the agreement to make 
`publish-summary: true`
the default selection.

[![cron example](https://github.com/cypress-io/github-action/workflows/example-cron/badge.svg?branch=master)](.github/workflows/example-cron.yml)

### Suppress test summary

> 

Reviewer: I suggest to add a sentence:

"The default test summary can be suppressed by using the parameter `publish-summary` and setting its value to `false`."

to introduce your explanation.

### Suppress test summary

> The default test summary can be suppressed by using the parameter publish-summary and setting its value to false.

Reviewer: ```suggestion
The default test summary can be suppressed by using the parameter `publish-summary` and setting its value to `false`.
```

The parameters are code so should be surrounded by backticks.
